web/src/pages/apd/key-state-personnel/ApdStateProfileMedicaidOffice.js

```diff
@@ -19,6 +21,41 @@ import { selectKeyStatePersonnel } from '../../../redux/selectors/apd.selectors'
 import { selectState } from '../../../redux/reducers/user';
 import { STATES } from '../../../util';
 
+import Joi from 'joi';
+
+const keyMedicaidSchema = Joi.object({
```

Comment:
could you put this in the new common/schema file, you can see how it's referenced in ContractorResourcesForm

Reply:
Yes! I knew I forgot something.